tech.json

```diff
@@ -485,7 +490,7 @@
                 {
                   "name": "canStaggeredWalljump",
                   "requires": [ "canConsecutiveWalljump" ],
-                  "note" : "Controlling wall jump height and positioning to dodge enemies or wait for something."
+                  "note" : "Controlling wall jump height and positioning to dodge enemies, wait for something, or set up Samus' wall jump positioning."
```

Comment:
The new phrase could probably use some clarification. "Controlling positioning to ... set up Samus' wall jump positioning" sounds a little circular?

Reply:
We had an example where we wanted to add this tech to it in order to move the tech to the right difficulty tier. If this description is not accurate in the way the logic is currently implemented, then it shouldn't be added in this PR - I'll remove it for now.